specs/dwritecore/DWriteCore.md

```diff
@@ -78,33 +78,275 @@ with interoperability issues.
+    by the font.
+
+However, migrating to the typographic font family model raises potential compatibility issues. The
+next section describes these issues and how they are mitigated by the hybrid font selection model.
+
+## Hybrid Font Selection Model
+
+The hybrid font selection model combines aspects of the weight-stretch-style and typographic font family
+models. This preserves the advantages of the typographic model while mitigating the following potential 
+compatibility issues:
+
+  - Some older fonts cannot be selected unambiguously using only the typographic family name and
+    axis values.
+
+  - Existing documents might refer to fonts by WWS family name or RBIZ family name. Users might also 
+    expect to use WWS and RBIZ family names. For example, a document might specify "Sitka Subheading" 
+    (a WWS family name) instead of "Sitka" (a typographic family name).
+    
+  - A library or framework might adopt the typographic font family model to take advantage of automatic
+    optical sizing, but not provide an API for specifing arbitrary axis values. Even if a new API is
+    provided, the framework might need to work with existing applications that specify only weight,
+    stretch, and style parameters.
+
+The compatibility issue with older fonts arises because the concept of typographic family name predates
+the concept of font axis values, which were introduced along with variable fonts in OpenType 1.8. Before
+OpenType 1.8, the typographic family name merely expressed the designer's intent that a set fonts were
+related, but with no guarantee that those fonts could be programmatically differentiated based on their
+properties. As a hypothetical example, suppose all of the following fonts have the typographic family
+name "Legacy":
+
+File                    | WWS Family   | Weight    | Stretch   | Style  | Typo Family   | wght  | wdth  | ital  | slnt
+------------------------|--------------|-----------|-----------|--------|---------------|-------|-------|-------|------
+Legacy-Regular.ttf      | Legacy       | 400       | 5         | 0      | Legacy        | 400   | 100   | 0     | 0
+Legacy-Bold.ttf         | Legacy       | 700       | 5         | 0      | Legacy        | 700   | 100   | 0     | 0
+Legacy-Black.ttf        | Legacy       | 900       | 5         | 0      | Legacy        | 900   | 100   | 0     | 0
+Legacy-Soft.ttf         | Legacy Soft  | 400       | 5         | 0      | Legacy        | 400   | 100   | 0     | 0
+Legacy-SoftBold.ttf     | Legacy Soft  | 700       | 5         | 0      | Legacy        | 700   | 100   | 0     | 0
+Legacy-SoftBlack.ttf    | Legacy Soft  | 900       | 5         | 0      | Legacy        | 900   | 100   | 0     | 0
+
+The "Legacy" typographic family has three weights and each weight has regular and "Soft" variants.
+However, these fonts predate OpenType 1.8, so there is no "SOFT" axis, or any other axis that cannot
```

Comment:
You might insert, "If these were new fonts, they could be implemented as declaring a SOFT design axis."